README.md

```diff
@@ -35,15 +35,14 @@ dnf install -y git rust rust-std-static cargo rustfmt clippy postgresql
 ``` bash
 git clone https://github.com/pgopr/pgopr.git
 cd pgopr
-cargo build
-cd target/debug
+make install
 kind create cluster
-./pgopr install
-./pgopr provision primary
+pgopr install
+pgopr provision primary
 kubectl port-forward postgresql-XYZ 5432:5432
 psql -h localhost -p 5432 -U myuser --password mydb
-./pgopr retire primary
-./pgopr uninstall
+pgopr retire primary
+pgopr uninstall
 kind delete cluster
```

Comment:
You need to explain that the binary it is installed in `/usr/local/bin`, and that the directory need to be in `$PATH`.

Make a paragraph after the instructions with this